src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2793,14 +2793,24 @@ def update_linux_profile(self, mc: ManagedCluster) -> ManagedCluster:
 
         if ssh_key_value:
             if mc.linux_profile is None:
-                raise InvalidArgumentValueError("Updating the cluster with no ssh key set at creation is not supported")
-            mc.linux_profile.ssh = self.models.ContainerServiceSshConfiguration(
-                public_keys=[
-                    self.models.ContainerServiceSshPublicKey(
-                        key_data=ssh_key_value
+                mc.linux_profile = self.models.ContainerServiceLinuxProfile(
```

Comment:
Will rp handle the backfill if admin_username not set previsouly? If so, may skip the backfill here

Reply:
If I don't set "admin_username" here, I'll get an error like:

```
  File "/home/binxi/.azure/cliextensions/aks-preview/azext_aks_preview/managed_cluster_decorator.py", line 2796, in update_linux_profile
    mc.linux_profile = self.models.ContainerServiceLinuxProfile(
TypeError: ContainerServiceLinuxProfile.__init__() missing 1 required keyword-only argument: 'admin_username'
```